internal/commands/cnab.go

```diff
@@ -161,3 +184,51 @@ func resolveBundle(dockerCli command.Cli, name string, pullRef bool, insecureReg
 	}
 	return nil, fmt.Errorf("could not resolve bundle %q", name)
 }
+
+func requiredClaimBindMount(c claim.Claim, targetContextName string, dockerCli command.Cli) (bindMount, error) {
+	var specifiedOrchestrator string
+	if rawOrchestrator, ok := c.Parameters["docker.orchestrator"]; ok {
+		specifiedOrchestrator = rawOrchestrator.(string)
+	}
+
+	return requiredBindMount(targetContextName, specifiedOrchestrator, dockerCli)
+}
+
+func requiredBindMount(targetContextName string, targetOrchestrator string, dockerCli command.Cli) (bindMount, error) {
+	if targetOrchestrator == "kubernetes" {
+		return bindMount{false, ""}, nil
+	}
+
+	socketPath := "/var/run/docker.sock"
+	// TODO:smarter handling of default context required
+	if targetContextName == "" {
+		return bindMount{true, socketPath}, nil
+	}
+
+	ctxMeta, err := dockerCli.ContextStore().GetContextMetadata(targetContextName)
+	if err != nil {
+		return bindMount{false, ""}, err
+	}
+	dockerCtx, err := command.GetDockerContext(ctxMeta)
+	if err != nil {
+		return bindMount{false, ""}, err
+	}
+	if dockerCtx.StackOrchestrator == command.OrchestratorKubernetes {
+		return bindMount{false, ""}, nil
+	}
+	dockerEndpoint, err := docker.EndpointFromContext(ctxMeta)
+	if err != nil {
+		return bindMount{false, ""}, err
+	}
+
+	host := dockerEndpoint.Host
+	if strings.HasPrefix(host, "unix://") {
+		socketPath = strings.TrimPrefix(host, "unix://")
```

Comment:
is the lack of an `npipe://` alternative expected here?

Reply:
I mixed up `host` and `socketPath` so I see now that `npipe://foo` as input does return `{true, "/var/run/docker.sock"}` as expected (not `{true, "npipe://foo"}` like I thought).